admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -194,9 +200,26 @@ export default class OrderProductRenderer {
   }
 
   toggleProductAddNewInvoiceInfo() {
-    $(OrderViewPageMap.productAddNewInvoiceInfo).toggleClass(
-      'd-none',
-      $(OrderViewPageMap.productAddInvoiceSelect).val() === 0,
-    );
+    if ($(OrderViewPageMap.productAddInvoiceSelect).val() === 0) {
```

Comment:
Need to confirm it's correct (replace `toggleClass()` by `removeClass('d-none')` / `addClass('d-none')`

Reply:
use the `toggleClass` instead